cmd/zc_traverser_local.go

```diff
@@ -316,18 +315,14 @@ func newLocalTraverser(fullPath string, recursive bool, followSymlinks bool, inc
 }
 
 func cleanLocalPath(localPath string) string {
-	normalizedPath := path.Clean(replacePathSeparators(localPath))
+	normalizedPath := path.Clean(consolidatePathSeparators(localPath))
 
-	// detect if we are targeting a network share
-	if strings.HasPrefix(localPath, "//") || strings.HasPrefix(localPath, `\\`) {
-		// if yes, we have trimmed away one of the leading slashes, so add it back
-		normalizedPath = common.AZCOPY_PATH_SEPARATOR_STRING + normalizedPath
-	} else if len(localPath) == 3 && (strings.HasSuffix(localPath, `:\`) || strings.HasSuffix(localPath, ":/")) ||
+	// detect if we are targeting a drive (ex: either C:\ or C:)
+	// note that on windows there must be a slash in order to target the root drive properly
+	// otherwise we'd point to the path from where AzCopy is running (if AzCopy is running from the same drive)
+	if len(localPath) == 3 && (strings.HasSuffix(localPath, `:\`) || strings.HasSuffix(localPath, ":/")) ||
 		len(localPath) == 2 && strings.HasSuffix(localPath, ":") {
-		// detect if we are targeting a drive (ex: either C:\ or C:)
-		// note that on windows there must be a slash in order to target the root drive properly
-		// otherwise we'd point to the path from where AzCopy is running (if AzCopy is running from the same drive)
-		normalizedPath += common.AZCOPY_PATH_SEPARATOR_STRING
+		normalizedPath += common.OS_PATH_SEPARATOR
```

Comment:
>normalizedPath += common.OS_PATH_SEPARATOR [](start = 2, length = 42)

so for C:\, we'd append another '\'?

Reply:
(note: edited your comment as \\ was formatted incorrectly, made readability hard.)

But yes, we would.